docs/source/conf.py

```diff
@@ -27,14 +27,14 @@
 TMP_FOLDER = Path(__file__).parents[2] / "tmp"
 
 if os.environ.get("READTHEDOCS_VERSION_TYPE") == "tag":
-    SUBSTRA_VERSION = "0.41.0"
+    SUBSTRA_VERSION = "0.42.0"
     TOOLS_VERSION = "0.20.0"
-    SUBSTRAFL_VERSION = "0.33.0"
+    SUBSTRAFL_VERSION = "0.34.0"
 
 else:
-    SUBSTRA_VERSION = "main"
-    TOOLS_VERSION = "main"
-    SUBSTRAFL_VERSION = "main"
+    SUBSTRA_VERSION = "release/0.42.0"
+    TOOLS_VERSION = "0.20.0"
+    SUBSTRAFL_VERSION = "release/0.34.0"
```

Comment:
Change back to `main` before release

Reply:
You need to put 0.42.0rc1 and 0.34.0rc1 to test the release. SubstraFL and Substra are officially release in the last step :)